src/components/__test__/ImagesLength.test.js

```diff
@@ -0,0 +1,18 @@
+import { render, screen } from '@testing-library/react';
+import ImagesLength from '../ImagesLength';
+
+beforeEach(() => {
+    render(<ImagesLength imagesLength={2} />);
+})
+
+describe('check images count', () => {
+    it('check images count by passing props', () => {
+        const checkImageCount = screen.getByTestId(/img-count/i)
+        expect(checkImageCount.textContent).toBe('Found 2 Images')
+    });
+
+    it('negative test', () => {
```

Comment:
The description should explain what check you are eventually doing rather than a generic message.